core/src/qrsignatures.c

```diff
@@ -0,0 +1,43 @@
+#include <unistd.h>
+#include <stdbool.h>
+#include <string.h>
+#include <stdio.h>
+
+#include "ecdsa.h"
+#include "curves.h"
+#include "memzero.h"
+#include "nist256p1.h"
+#include "qrsignatures.h"
+#include "log.h"
+#include "config.h"
+
+static const uint8_t QR_PUBKEY[65] = _QR_PUBKEY;
+bool check_qrsignature(const uint8_t * const data, const size_t data_len, const uint8_t * const signature){
```

Comment:
The `const` qualifier on `data_len` is unnecessary, because it's not a pointer type.

Reply:
🤦 . right. removing.